docs/_includes/search-toggle.html

```diff
@@ -0,0 +1,5 @@
+<div class="search-icon">
+    <a href="javascript:void(0)" onclick="openSidebar()" id="toggle-search-open">
```

Comment:
I think the href is not necessary here
```suggestion
    <a onclick="openSidebar()" id="toggle-search-open">
```